docs/tutorials/working_with_the_ensemble.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<p>The object table is a bit boring...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Could we just add few columns for the object table? Or we cannot actually do it with `from_source_dict`?

Reply:
Using `from_pandas` felt the cleanest way to me

`from_source_dict` just resolves to calling `from_dask_dataframe` with [`object_frame=None`](https://github.com/lincc-frameworks/tape/blob/1d97c55111e9084434675fabd28674676f56e9dc/src/tape/ensemble.py#L1428) and then we just [generate an empty Object table](https://github.com/lincc-frameworks/tape/blob/1d97c55111e9084434675fabd28674676f56e9dc/src/tape/ensemble.py#L1096)